src/launcher.ts

```diff
@@ -11,9 +11,7 @@ import type { Capabilities } from '@wdio/types'
 import { HttpsProxyAgent } from 'hpagent'
 
 import startServer from './server/index.js'
-import {
-    fileExist, directoryExists, isMultiremote, isChrome
```

Comment:
We should remove `isMultiremote` and `isChrome` entirely then, since these functions aren't used anymore.